full-source-install.yml

```diff
@@ -0,0 +1,35 @@
+# ansible-playbook -i hostname-or-ip, -u root --ask-become-pass full-source-install.yml
+---
+- hosts: all
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+    pulp_source_dir: "/pulp_source/pulp"
+    pulp_git_repo: https://github.com/pulp/pulp.git
+    pulp_secret_key: "unsafe_default"
+    pulp_default_admin_password: admin
+    pulp_debug: true
```

Comment:
please add documentation of this new variable. https://github.com/pulp/ansible-pulp3/blob/master/roles/pulp3/README.md